src/managers/GuildMemberManager.js

```diff
@@ -237,6 +237,8 @@ class GuildMemberManager extends CachedManager {
 
     // Clone the data object for immutability
     const _data = { ...data };
+    _data.communication_disabled_until = new Date(Date.now() + _data.communicationDisabledUntil * 1000).toISOString();
+    if (_data.communicationDisabledUntil < 0) throw new Error('The seconds have to be positive');
```

Comment:
Seeing as the API supports an ISO string, you could make this accept a Date or a number which could be a UNIX timestamp in ms or the amount of seconds from now. You'd have to do more elaborate checks though. Also, you're currently passing a camelCased value to the API, which you should delete instead. Lastly, the error should be created in Messages.js and the message can be improved a lot.